### DISCUSSION TITLE
intermittent modbus connection FZ94 Pro Lab

### DISCUSSION THREAD

**Discussion by manuelserafin** [artisan-roaster-scope/artisan#1735] [Q&A] [answered]
Hi,
I’m experiencing the exact same issue as discussed in an older thread [https://github.com/artisan-roaster-scope/artisan/discussions/846], but with an FZ94 Pro Lab roaster and Artisan version 3.0.2. I’m not using any additional interface—just the USB connection that came with the machine, plugged directly into my computer. I’ve already checked the timeout settings and sample rate, but they don’t seem to have any effect on the problem.

Could you let me know if there have been any recent changes in how modbus communication is handled in version 3.0.2, or if there’s anything else I might try? Thanks in advance!

**Comment by MAKOMO** [comment, marked as answer]
Yes, see Issue #1694.

and got fixed in the current [continuous builds](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous). Either install that or wait for the next regular release scheduled for this November.
